### ISSUE TITLE
Discussions on sklearn

### ISSUE THREAD

**Issue by bonartm** [stefanfroth/nmf_extension#5]
Hi Stefan (I saw your comment https://github.com/spicedacademy/datascience_source/issues/223#issuecomment-737381613),

for inspiration you maybe also want to have a look at these two issues/ pull requests from the sklearn project:

    scikit-learn/scikit-learn#8447
    scikit-learn/scikit-learn#8474

I think, the idea is, that you modify the loss function in such a way that it ignores missing entries.
